pdf-service/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is this file part of the bug fixes? Doesn't seem related... Also, did we not remove the node dependency for PDF service?